src/test/java/com/wikia/webdriver/elements/mercury/pages/CategoryPage.java

```diff
@@ -39,174 +30,66 @@ public CategoryPage() {
     navigate = new Navigate();
   }
 
-  public CategoryPage navigateToPageWithArticleAndWithMembersFromUrl() {
-    navigate.toPage(MercurySubpages.CATEGORY_WITH_ARTICLE_AND_WITH_MEMBERS);
-
-    articleContainerIsVisible();
-    categorySectionsContainerIsVisible();
-
-    return this;
-  }
-
-  public CategoryPage navigateToPageWithArticleAndWithoutMembersFromUrl() {
-    navigate.toPage(MercurySubpages.CATEGORY_WITH_ARTICLE_AND_WITHOUT_MEMBERS);
-
-    articleContainerIsNotPresent();
-    categorySectionsContainerIsNotPresent();
-    verifyErrorPage();
-
-    return this;
-  }
-
-  public CategoryPage navigateToPageWithoutArticleAndWithMembersFromUrl() {
-    navigate.toPage(MercurySubpages.CATEGORY_WITHOUT_ARTICLE_AND_WITH_MEMBERS);
-
-    articleContainerIsNotPresent();
-    categorySectionsContainerIsVisible();
-
-    return this;
-  }
-
-  public CategoryPage loadMoreMembersForSection(String name) {
-    navigateThroughSection(name, loadMoreButton);
+  public CategoryPage open(String categoryName) {
+    this.navigate.toPage(String.format("%s%s", URLsContent.WIKI_DIR, categoryName));
 
     return this;
   }
 
-  public CategoryPage loadPreviousMembersForSection(String name) {
-    navigateThroughSection(name, loadPreviousButton);
 
-    return this;
-  }
+  public ArticlePage navigateToCategoryMemberPage() {
+    WebElement member = driver.findElement(categoryMembers);
+    String memberName = member.getText();
 
-  public CategoryPage navigateToCategoryMemberPage(String name) {
-    WebElement member = driver.findElement(By.xpath(
-        String.format("//a[contains(text(), \"%s\")]", name))
-    );
+    wait.forXMilliseconds(1500);
```

Comment:
What's this for? Shouldn't `new CategoryPage().open(categoryName)` make sure that the page is ready to be tested? Also, we query the DOM before waiting, what happens during these 1500 ms?